README.md

```diff
@@ -10,76 +10,53 @@ We have designed rCTF with the following attributes in mind:
 * simplicity
 * "modernness" (no PHP)
 
-## Installation
+## Getting Started
 
-### Automatic
+### Installation
 
-The automatic installation script works on Debian-based distributions and Arch Linux. It depends on `curl`.
+#### Automatic
+The automatic installation script works on Debian-based distributions and Arch Linux. It depends on `curl`. Feel free to [read the script](https://get.rctf.redpwn.net/) before running this command as root (it's fairly short).
 
-```bash
+```
 # curl https://get.rctf.redpwn.net | sh
 ```
 
-### Manual
+#### Manual
 
-#### Environment
+For instructions on manual installations and deployments, see the [wiki page](https://github.com/redpwn/rctf/wiki/Manual-Deployment).
 
-You should copy the `.env.example` to `.env`. 
+### Management
 
-```bash
-$ cp .env.example .env
-```
+The rCTF CLI management tool [install/rctf.py](`rctf`) (installed in `/usr/bin/` by default by the automatic installation script) makes management of your rCTF installation simple.
 
-Note that the `RCTF_TOKEN_KEY` should be a base64 encoded string of length 32 bytes. You can generate one with the following command.  
+#### Starting rCTF
 
-```bash
-$ openssl rand -base64 32
-j9qpC+J9lrvx/F7uQ9wGKawhzfPyPb1aM+JPeLfsFX8=
 ```
-
-#### Database
-
-The application is built on a PostgreSQL database. You should add the appropriate connection string in `.env`. Then run the following command to setup the database. 
-
-```bash
-$ yarn run migrate up
+# rctf up
 ```
 
-## Development
-
-### Backend
-
-For hot reloading, use nodemon.
+#### Stopping rCTF
 
-```bash
-$ yarn dev
 ```
-
-There is a precommit hook that will block commits that fail to meet the style guide. To fix style errors, use standard. 
-
-```bash
-$ yarn lint --fix
+# rctf down
 ```
 
-### Frontend
-
-The frontend is built on [Preact](https://preactjs.com/) and [Cirrus](https://spiderpig86.github.io/Cirrus/). 
-
-All of the frontend code is in `/client`. To automatically watch the directory for changes, use the `watch` command. 
+#### Upgrading rCTF
 
-```bash
-$ yarn watch
+```
+# rctf update
 ```
 
-## Deployment
+#### Deploying Challenges to rCTF
 
-### Problem Data
+##### Using rDeploy
 
-To build, simply copy the exported directory from [rDeploy](https://github.com/redpwn/rdeploy) to `config.rDeployDirectory` (defaults to `.rdeploy`). 
+If you use [rDeploy](https://github.com/redpwn/rdeploy), you can automatically import the problem data using the below command:
 
-```bash
-$ cd rdeploy
-$ python main.py build
-$ cp -r export ../rctf/.rdeploy
 ```
+# rctf deploy /path/to/rdeploy/directory/
```

Comment:
Maybe this should be `rctf deploy /path/to/problem/directory` and we clone rdeploy for them